app/models/location.rb

```diff
@@ -13,7 +13,7 @@ def self.primary(city_name = ENV['PRIMARY_CITY_NAME'])
 
   def weather
     response = Clients::DarkskyClient.forecast(self).dup
-    response['solarcycles'] = solarcycles
+    response['solarcycles'] = solarcycles.after_beginning_of_yesterday.all.to_a
```

Comment:
What is the benefit of doing `.all.to_a` ? Wouldn't `solarcycles.after_beginning_of_yesterday` already return a shape that can be indexed?

Reply:
Needed `.all.to_a` to fix the graphql gem not working with the active record proxy record. Probably should split these out into two separate commits.